File: 3. Conditional Loops/1. Cpp/1. Basic/volumeofacone.cpp
#include <bits/stdc++.h>
using namespace std;
#define ll long long
#define FIO ios_base::sync_with_stdio(false), cin.tie(0), cout.tie(0)
#define pi 3.14
#define endl "\n"

int main() {
	FIO;
	freopen("input.txt", "r", stdin);
	freopen("output.txt", "w", stdout);
	double radius, height;
	cin >> radius >> height;
	double length = height / 3;
	double circle = pi * (radius * radius);
	double volume = (circle * length);
	if (radius < 0 || height < 0) {
		cout << "VOLUME CANNOT BE NEGATIVE" << endl;
	}
	else {
		cout << "Volume is: " << volume << endl;
	}
	return 0;
}

File: 3. Conditional Loops/2. Java/1. Basic/volumeofacone.java
import java.util.Scanner;
public class volumeofacone {
    private static final double pi = 3.1416;
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the radius: ");
        double radius = sc.nextDouble();
        System.out.print("Enter the height: ");
        double height = sc.nextDouble();
        double length = height/3;
        double circle = pi * (radius * radius);
        double volume = (circle * length);
        if (radius < 0 || height < 0){
            System.out.print("VOLUME CANNOT BE NEGATIVE ");
        }
        else{
            System.out.print("VOULME IS:  " + volume);
        }
        sc.close();
    }
}
